src/lib/eina/eina_sched.c

```diff
@@ -17,68 +17,22 @@
  */
```

Comment:
Will you do an eina_sched_{win32,posixe}.c?

Reply:
Not for a while

Reply:
yes